docs/scanners.md

```diff
@@ -172,6 +172,7 @@ Scan rules:
 40043   Log4Shell (CVE-2021-44228, CVE-2021-45046)
 40044   Exponential Entity Expansion (Billion Laughs Attack)
 40045   Spring4Shell (CVE-2022-22965)
+40046   Server Side Request Forgery
```

Comment:
Should the ID be in the 2XXXX series instead?

Reply:
🤷

Reply:
The ranges/series no longer indicate anything.

Reply:
Should we remove those notes?